src/dss/main.py

```diff
@@ -30,5 +31,28 @@ def initialize_command(kubeconfig: str) -> None:
     initialize(lightkube_client=lightkube_client)
 
 
+# FIXME: remove the `--kubeconfig`` option
+# after fixing https://github.com/canonical/data-science-stack/issues/37
+@main.command(name="remove-notebook")
+@click.option(
+    "--name",
```

Comment:
I think the name of the notebook should be required. What if not provided ? Should it remove all notebooks ?

Reply:
good point

Reply:
done